modules/mono/glue/GodotSharp/GodotSharp/Core/Rect2.cs

```diff
@@ -171,6 +171,26 @@ public readonly Vector2 GetCenter()
             return _position + (_size * 0.5f);
         }
 
+        /// <summary>
+        /// Returns the support point in a given direction.
+        /// This is useful for collision detection algorithms.
+        /// </summary>
+        /// <param name="dir">The direction to find support for.</param>
+        /// <returns>A vector representing the support.</returns>
+        public readonly Vector2 GetSupport(Vector2 dir)
```

Comment:
This method doesn't seem to be exposed to GDScript.

https://github.com/godotengine/godot/blob/3c0eaec39f62ba1d86ba12abdf3f95779e608c25/core/variant/variant_call.cpp#L1839-L1855